index.py

```diff
@@ -31,8 +44,23 @@ def home():
     return render_template('enter.html', library=library)
 
 
-@app.route('/index')
+@app.route('/index', methods=["GET", "POST"])
 def index():
+    if request.method == "POST":
+        if 'file' not in request.files:
+            flash('No file part')
+            return redirect(request.url)
+        file = request.files['file']
+        if file.filename == '':
+            flash('No selected file')
+            return redirect(request.url)
+        if file and allowed_file(file.filename):
+            filename = secure_filename(file.filename)
+            file.save(os.path.join(app.config['UPLOAD_FOLDER'], filename))
+            import_service = FileImport()

```

Comment:
I woluld like to use import like this:
```
import_service = FileImport(app.config['UPLOAD_FOLDER'])
import_service.process_file(file, filename)
```
It allows us to remove **os** dependency from index.py